app/app/Http/Controllers/PasswordRecoveryController.php

```diff
@@ -36,6 +36,21 @@ public function sendEmail(Request $request)
             return view('pages.password_recovery.unmatched_email');
         }
 
+        if( $matching_user->email_verification_time == null) {
```

Comment:
This if-statement breaks the code styling tests.  I'm about to make a separate pull request to fix the problem.

Please run "composer all" every time you make a pull request or update one.